game/scripts/npc/abilities/queenofpain_shadow_strike.txt

```diff
@@ -39,12 +39,12 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "strike_damage"                                   "50 75 100 125 250 375"
+        "strike_damage"                                   "75 100 125 150 250 400" //OAA
       }
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "duration_damage"                                 "30 50 70 90 150 375"
+        "duration_damage"                                 "40 60 80 100 150 400" //OAA
```

Comment:
Don't change non-OAA levels for duration dmg because people don't have enough dispels early-mid game to remove it. Put 30/50/70/90/180/360. 360 seems like a nerf but its fine because of the change above.

Reply:
I would like to defend this a bit actually. I think a 10 damage increase isn't going to kill anyone and is a smalll buff to qop. and the change to max level doesn't reallly matter because it's a level 6 ability and people should have dispels at that point.

Reply:
I'll revert the early levels back but to compromise can I make level 5, 200?

Reply:
180/360 is enough, cd is 4 seconds afterall and strike damage is more important.